setup.py

```diff
@@ -14,6 +14,23 @@
 
 from setuptools import setup
 
+# Figure out if the system already has a supported Crypto library
```

Comment:
Not sure if you'll like this, but I wanted a way to not force M2Crypto to be installed in my environment if I had an alternative crypto library that was supported (pycryptodome). Happy to live with this not making it into master.

Reply:
nah, but I wonder if there's an OR option in setuptools?

Reply:
Couldn't find an OR option in setuptools. Most just perform logic like this in the actual setup.py.